noir_stdlib/src/meta/expr.nr

```diff
@@ -669,7 +669,12 @@ comptime fn new_unary_op(op: UnaryOp, rhs: Expr) -> Expr {
 
 comptime fn new_unsafe(exprs: [Expr]) -> Expr {
     let exprs = join_expressions(exprs, quote { ; });
-    quote { unsafe { $exprs }}.as_expr().unwrap()
+    quote { unsafe { 
+        //@safety: generated by macro
+        $exprs 
+        }}
```

Comment:
```suggestion
    }}
```
Are these leading spaces from the formatter?

Reply:
I would have guess yes, but no, the formatter does not override them. It's a bit weird because it is not aligned with the next line however.